react/features/overlay/components/AbstractPageReloadOverlay.js

```diff
@@ -269,6 +279,7 @@ export function abstractMapStateToProps(state: Object) {
 
     return {
         isNetworkFailure: Boolean(configError || connectionError),
-        reason: (configError || connectionError || conferenceError).message
+        reason: (configError || connectionError || conferenceError).message,
+        details: connectionError ? connectionError.details : {}
```

Comment:
I wonder if it's better in this case to actually return undefined for details and use "defaultProps" instead. I don't know about the underlying implementation but maybe doing so would prevent unnecessary re-renders because a new object is being set for the prop details.

Reply:
Also, the comment for abstractMapStateToProps needs to be updated because the return value has changed.